src/server/command_registry.cc

```diff
@@ -119,8 +119,9 @@ void CommandRegistry::Init(unsigned int thread_count) {
 CommandRegistry& CommandRegistry::operator<<(CommandId cmd) {
   string k = string(cmd.name());
 
+  const bool is_sub_command = cmd.IsSubCommand();
+
```

Comment:
The code below will get refactor at some point. We can simplify, as we slowly migrate to treating subcommands as full commands